setup/index.js

```diff
@@ -0,0 +1,99 @@
+const path = require('path');
+const core = require('@actions/core');
+const tc = require('@actions/tool-cache');
+const { getDownloadObject, getDownloadPluginObject, addPluginCert, versionCheck } = require('./lib/utils');
+const { get } = require('http');
+const fs = require('fs');
+const mv = require('mv');
+const { version } = require('os');
+const execSync = require('child_process').execSync;
+
+async function setup() {
+  try {
+    // Get version of tool to be installed
+    const version = core.getInput('version'); //"0.9.0-alpha.1";
+    const keyName = core.getInput('key_name');
+    
+    // Check if the NotationCli version is supported
+    versionCheck(version);
+
+    // Check if the NotationCli version is supported
+    versionCheck(version);
```

Comment:
Do we need to check the version twice?
```suggestion
    // Check if the NotationCli version is supported
    versionCheck(version);
```

Reply:
Nope, that was a typo. :) Removed it.